tests/integration/dialog-extension/dialog_events.js

```diff
@@ -93,11 +93,11 @@ QUnit.asyncTest( "Clicking dialog 'Close' button renders it unclickable", functi
 
 	$.testHelper.pageSequence( [
 		function() {
-			$.mobile.changePage( $( "#mypage" ) );
+			$( ".ui-pagecontainer" ).pagecontainer( "change", $( "#mypage" ) );
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?